### ISSUE TITLE
Unable to open Dev Tools when opening a 400mb notebook with plots

### ISSUE THREAD

**Issue by DonJayamanne** [microsoft/vscode#211065]
Testing #210949

No time line entries for large notebooks even with
* Auto save enabled
* Manually saving notebooks

**Comment by amunger** [comment]
looks like this is expected for large files
![image](https://github.com/microsoft/vscode/assets/2019016/6dd799f8-5d34-4fa0-9a5c-08ac29c837be)

**Comment by DonJayamanne** [comment]
Weird, I didn't see such a message.
Let me try again,

**Comment by DonJayamanne** [comment]
Yes, this is the message I get,
All i did was open a (existing) large notebook and had timeline view open.

![Screenshot 2024-04-24 at 06 04 49](https://github.com/microsoft/vscode/assets/1948812/2ee18d5a-6b07-410f-a4c6-b64302bfdb32)
